src/networkwatcher.cpp

```diff
@@ -11,6 +11,8 @@
 #include "platforms/dummy/dummynetworkwatcher.h"
 #include "settingsholder.h"
 
+#include <QMetaEnum>
+
 #ifdef MVPN_WINDOWS
```

Comment:
If every single platform now has a `NetworkWatcher` would could move this into a factory class, like we have done with other platform-specific classes (eg: `PingSenderFactory`)